### DISCUSSION TITLE
Fleet ？

### DISCUSSION THREAD

**Discussion by BTMuli** [Arondight/Adachi-BOT#948] [Ideas]
Jetbrains 的一个新 IDE，现在处于免费期，不知道体验如何，有试过的嘛(＊ﾟ◇ﾟ)

**Comment by mark9804** [comment]
#939

我的评价是一坨狗屎，不过有可能它只是对 vue 支持不太好

**Reply by BTMuli** [reply]
🤣
![1666752887344.jpeg](https://user-images.githubusercontent.com/72692909/197923621-a462bcaa-a325-429e-a0a3-2f18105b489d.jpeg)

**Reply by mark9804** [reply]
目前真不如VSC，内存泄漏不是关键（好吧也很关键），但是功能又少代码高亮又烂用户交互也烂内存还泄漏（没注意到爆 CPU），VSCode 看了做梦都能乐醒

**Comment by mark9804** [comment]
内存占用
<img width="264" alt="image" src="https://user-images.githubusercontent.com/9006264/198828419-458a2c52-f7c6-4614-a2f3-8022a7a1112a.png">

**Comment by mark9804** [comment]
<img width="267" alt="image" src="https://user-images.githubusercontent.com/9006264/205916294-eae65404-ee80-43c3-adda-22b30cccf885.png">
猜猜 5.2G 是什么？

猜对了，是两个版本的 fleet
<img width="280" alt="image" src="https://user-images.githubusercontent.com/9006264/205916488-b65d3846-03f6-4936-8ad7-e41ad340bc47.png">
